config.template

```diff
@@ -97,6 +97,14 @@ sstSubdirectory = SST
 sssSubdirectory = SSS
 mldSubdirectory = MLD
 
+# first and last year of SST climatology
+# values for preindustrial
+sstClimatologyStartYear = 1870
+sstClimatologyEndYear = 1900
+# alternative values for present day
+#sstClimatologyStartYear = 1990
+#sstClimatologyEndYear = 2011
```

Comment:
@milenaveneziani, no.  The model time is not used here at all.  Instead, years 1870 to 1900 are loaded from the observational data and then averaged together to make a climatology (without ever knowing anything at all about the simulation time).  Then, that climatology is compared with one made from the simulation.

Obviously, it would seem strangely redundant if you did have a simulation that covered, say, 1850 to 2000 and you said you wanted to make a climatology from 1870 to 1900 and you also had to say you wanted to compare with observation from 1870 to 1900, but that is what you would have to do with my suggested changes.

I agree that it would be much simpler to just have a single option instead -- `period = 'pre-industrial'` or `period = 'present-day'` and each observational climatology would be selected accordingly.  But, again, I think that's outside the scope of this PR.

What can I do to address these concerns *within* the scope of this PR?